### ISSUE TITLE
[Bug] mutation PlayerCreate player size 

### ISSUE THREAD

**Issue by jonahhb** [f1multiviewer/issue-tracker#472]
There used to be a problem with saving layouts where if you had your Windows display settings "Scale" set to anything but 100% the Players would all open in the wrong sizes. While this bug is fixed for the save layout function, it is not fixed for when a player is created with the mutation PlayerCreate in the API.

# Here is what happens: 
- https://discordapp.com/channels/982375740764983366/1025395943274721321/1208538327423389696 
- https://discordapp.com/channels/982375740764983366/1025395943274721321/1208538414572380300
### For reference here is what the 'swap feeds button' mentioned does with the mutation PlayerCreate:
- The important bit is that it gives the new player the bounds information of the old player but then the player ends up a different size when the Windows scale option in display settings is not set to 100%. See image:
![image](https://github.com/f1multiviewer/issue-tracker/assets/102438585/422b059e-2100-4005-823e-450b1383bedc)

# Here is how it was figured out that it is similar to (or the same as) the saving layouts bug:
- In short, it works correctly again when the Windows setting is turned to 100%. 
- https://discordapp.com/channels/982375740764983366/1025395943274721321/1209171870079787038
- https://discordapp.com/channels/982375740764983366/1025395943274721321/1209226198434062346
- https://discordapp.com/channels/982375740764983366/1025395943274721321/1209227312059388007
- https://discordapp.com/channels/982375740764983366/1025395943274721321/1209228598343630879

# Why it's still annoying: 
- Text size problems with the live timing window and having to switch it every race weekend. 
- https://discordapp.com/channels/982375740764983366/1025395943274721321/1209229136468643910

**Comment by LapsTimeOFF** [comment]
Hey, we have made some improvements. Is this issue still occurring?
